packages/react-components/src/utils/__tests__/text.test.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  getUniqueCommaStringWithSuffix,
+  getUniqueList,
+  appendSuffix,
+  getUniqueCommaString,
+} from '../index';
+
+describe('appendSuffix', () => {
+  it('can add a suffix to each value of the list', () => {
+    expect(appendSuffix('one', 'LAB')).toEqual('one LAB');
+  });
+});
+
+describe('getUniqueList', () => {
+  it('is case sensitive', () => {
```

Comment:
This test isn't required. We only need to test things we right code for. This is testing how things normally work.